clients/ethereum/client_test.go

```diff
@@ -204,6 +204,60 @@ func TestClient_GetBatch(t *testing.T) {
 		assert.Nil(t, batch)
 		assert.Equal(t, expectedErr, err)
 	})
+	t.Run("error while getting deposits", func(t *testing.T) {
+		c.clientWrapper = &bridgeTests.EthereumClientWrapperStub{
+			GetBatchCalled: func(ctx context.Context, batchNonce *big.Int) (contract.Batch, error) {
+				return contract.Batch{
+					Nonce:         batchNonce,
+					DepositsCount: 2,
+				}, nil
+			},
+			GetBatchDepositsCalled: func(ctx context.Context, batchNonce *big.Int) ([]contract.Deposit, error) {
+				return nil, expectedErr
+			},
+		}
+		batch, err := c.GetBatch(context.Background(), 1)
+		assert.Nil(t, batch)
+		assert.Equal(t, expectedErr, err)
+	})
+	t.Run("deposits mismatch - with 0", func(t *testing.T) {
```

Comment:
empty line before a test finish and a new `t.Run` ?

Reply:
all tests are written without breaks. In an IDE like goland, the tests are separated by the green run button anyway.